app/assets/stylesheets/components/_action-link.scss

```diff
@@ -0,0 +1,100 @@
+.app-c-action-link {
+  display: inline-block;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-align: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding-left: 50px;
+  min-height: 45px;
+  max-width: 460px;
+  background-image: image-url("covid-arrow.png"); // PNG fallback for SVG
+  background-image: image-url("covid-arrow.svg");
+  background-repeat: no-repeat;
+  background-position: 0 50%;
+  background-size: 37px auto;
+
+  @include govuk-media-query($until: tablet) {
+    max-width: 410px;
+    margin-bottom: govuk-spacing(8);
+  }
+
+  @include govuk-media-query($from: desktop) {
+    // IE10/11 specific - vertical align in flexbox is ignored
+    // add padding to action link container to improve arrow/text alignment on IE
+    @media screen and (min-width:0\0) {
+      padding-top: govuk-spacing(2);
+    }
+  }
+}
+
+.app-c-action-link__nowrap-text {
+  white-space: nowrap;
+}
+
+.app-c-action-link__link {
+  @include govuk-font(19, $weight: bold);
+  display: inline-block;
+  max-width: 100%;
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &:focus,
+  &:active {
+    text-decoration: none;
+    color: govuk-colour("black");
+  }
+}
+
+.app-c-action-link--dark-icon {
+  @include govuk-media-query($until: tablet) {
```

Comment:
Can we move these media queries to after the css rules defined on line 63+? We normally put css rules first, followed by modifiers and sub classes.